packages/component-library/src/components/table-and-list/mt-pagination/mt-pagination.spec.ts

```diff
@@ -115,5 +115,18 @@ describe("mt-pagination", () => {
 
       expect(wrapper.emitted()["change-current-page"][0]).toStrictEqual([7]);
     });
+
+    it("should disable the next page and last page button when total page value is zero", async () => {
```

Comment:
Please write the test using `@testing-library/vue`. You can see how it's done by opening the `mt-modal.spec.ts` file.